src/throttler.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from '@nestjs/common';
+import { ThrottlerStorage } from './throttler-storage.interface';
+
+@Injectable()
+export class ThrottlerStorageService implements ThrottlerStorage {
+  storage: Record<string, number[]> = {};
```

Comment:
```suggestion
  private storage: Record<string, number[]> = {};
```
(?)

Reply:
If it's enforced by the interface, consider creating a getter instead (that clones the underlying array)